packages/components/src/inputs/src/TextField.tsx

```diff
@@ -230,17 +229,14 @@ function TextField(props: TextFieldProps, ref: ForwardedRef<HTMLDivElement>) {
         </ClearContainerSlots>
     );
 
-    const childrenMarkup = composeRenderProps(children, prev => {
-        return (
-            <>
-                {label && <Label className={styles["hop-TextField__Label"]} isRequired necessityIndicator={necessityIndicator}>{label}</Label>}
-                {inputMarkup}
-                {prev}
-                {description && <HelperMessage className={styles["hop-TextField__HelperMessage"]}>{description}</HelperMessage>}
-                <ErrorMessage className={styles["hop-TextField__ErrorMessage"]}>{errorMessage}</ErrorMessage>
-            </>
-        );
-    });
+    const childrenMarkup = (
+        <>
+            {label && <Label className={styles["hop-TextField__Label"]} isRequired necessityIndicator={necessityIndicator}>{label}</Label>}
```

Comment:
what is the isRequired on the label for?

Reply:
it has to do with the label's necessity label/indicator